test/e2e/storage/testsuites/disruptive.go

```diff
@@ -47,11 +46,14 @@ func InitDisruptiveTestSuite() TestSuite {
 		},
 	}
 }
-
 func (s *disruptiveTestSuite) getTestSuiteInfo() TestSuiteInfo {
 	return s.tsInfo
 }
 
+func (s *disruptiveTestSuite) skipRedundantSuite(driver TestDriver, pattern testpatterns.TestPattern) {
+	skipVolTypePatterns(pattern, driver, map[testpatterns.TestVolType]bool{testpatterns.PreprovisionedPV: true})
```

Comment:
I'd prefer something like `newPatternMap(testpatterns.PreprovisionedPV)` instead of `map[testpatterns.TestVolType]bool{testpatterns.PreprovisionedPV: true}`

Or have two global consts (vars) for PreprovisionedPV and PreprovisionedPV+InlineVolume at the top and reuse them everywhere.

Reply:
done